src/types.db

```diff
@@ -57,7 +57,9 @@ df_inodes               value:GAUGE:0:U
 dilution_of_precision   value:GAUGE:0:U
 disk_allocation         value:GAUGE:0:U
 disk_capacity           value:GAUGE:0:U
+disk_discard            complete:DERIVE:0:U, merged:DERIVE:0:U, sectors:DERIVE:0:U, time:DERIVE:0:U
```

Comment:
Please don't combine multiple metrics / data sources into one "data set". Instead, use a single data source called "value" and put "complete", "merged", … into the `type_instance` field.

Reply:
I put them into the `type` field to align with other metrics in disk.c (almost all of them put them into `type`, e, g, "disk_ops", "disk_time"...)

Reply:
/* this feature */
[{"values":[0],"dstypes":["derive"],"dsnames":["value"],"time":1705908765.727,"interval":10.000,"host":"7643","plugin":"disk","plugin_instance":"nvme0n1","type":"disk_discard_merged","type_instance":""}]
write_log values:

/* other old metric */
[{"values":[94271,10518372],"dstypes":["derive","derive"],"dsnames":["read","write"],"time":1705908775.727,"interval":10.000,"host":"7643","plugin":"disk","plugin_instance":"nvme0n1","type":"disk_merged","type_instance":""}]